data/campaigns/Legend_of_Wesmere/scenarios/chapter5/23_End_of_War.cfg

```diff
@@ -60,7 +60,6 @@
     [side]
         side=2
 
-        no_leader=yes
```

Comment:
I can't quite remember, but most likely because of the `canrecruit=yes` and/or the `{PLAYABLE}` macro? Basically the idea is not to include keys for defining a leader unit when `no_leader=yes`, and not include `no_leader=yes` if there are keys defining a leader unit. Should I have removed the `canrecruit=yes` instead here?